packages/strapi-design-system/src/IconButton/IconButton.stories.mdx

```diff
@@ -95,6 +95,31 @@ IconButtons can be used within another component is a Group shape.
   </Story>
 </Canvas>
 
+### Icons as Children & aria-label
+
+IconButtons can take Icons as their children and can be fully accessible without a tooltip.
+
+<Canvas>
+  <Story name="children">
+    <Box padding={7}>
+      <Flex>
+        <IconButton onClick={() => console.log('edit')} aria-label="Edit">
```

Comment:
What do you think about updating some visible text instead? It is easier to understand for readers of the docs, I think.

Reply:
I'm not sure i understand what you're proposing here.

Reply:
I wanted to say that it could be nice to give visual feedback to somebody reading this documentation. Instead of logging these values, you could add them to a local store and render the value, to make it clear something is happening.